### PR TITLE
#1211 Postgres upper case schema fix

### PR DESCRIPTION
**Pull request by smohara**
Remove the quotes from a schema name returned by the doGetCurrentSchemaName  in PostgreSQLDbSupport.

### CODE REVIEW

File: flyway-core/src/test/java/org/flywaydb/core/internal/dbsupport/postgresql/PostgreSQLDbSupportSmallTest.java [hunk 1]
```diff
@@ -26,4 +26,14 @@ public void doQuote() {
         assertEquals("\"abc\"", dbSupport.doQuote("abc"));
         assertEquals("\"a\"\"b\"\"c\"", dbSupport.doQuote("a\"b\"c"));
     }
+
+    @Test
+    public void undoQuote() {
+        PostgreSQLDbSupport dbSupport = new PostgreSQLDbSupport(null);
+        assertEquals("abc", dbSupport.unQuote("\"abc\""));
+        assertEquals("abc", dbSupport.unQuote("abc\""));
```
Comment (axelfontaine) [inline flyway-core/src/test/java/org/flywaydb/core/internal/dbsupport/postgresql/PostgreSQLDbSupportSmallTest.java]: Why is this and all other cases from line 34 to 37 necessary?

File: flyway-core/src/main/java/org/flywaydb/core/internal/dbsupport/postgresql/PostgreSQLDbSupport.java [hunk 2]
```diff
@@ -119,6 +119,14 @@ public String doQuote(String identifier) {
         return "\"" + StringUtils.replaceAll(identifier, "\"", "\"\"") + "\"";
     }
 
+    public String unQuote(final String quoted) {
```
Comment (axelfontaine) [inline flyway-core/src/main/java/org/flywaydb/core/internal/dbsupport/postgresql/PostgreSQLDbSupport.java]: Can you make this package-private and add JavaDoc?

**Comment by axelfontaine** [comment]
I am having doubts about this fix. How does it deal with multiple schemas in the search path?

**Comment by smohara** [comment]
It doesn't work well when multiple schema are returned. 
The method this is used in is called doGetCurrentSchemaName, which suggests it will return a single schema name. Prior to my change it would have returned multiple schemas. I assume we want to treat the first element in the search path as the current schema?

**Comment by axelfontaine** [comment]
It is actually there to store the current schema of a connection and restore it afterwards. For PG this actually means storing the search_path and restoring it. The name is misleading for PG but correct for the other DBs.

**Comment by axelfontaine** [comment]
Closing due to lack of response from original reporter

**Pull request closed without merging**